Directory.Packages.props

```diff
@@ -1,20 +1,27 @@
-<Project>
+<Project>
+
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+    <!-- Using multiple feeds isn't supported by Maestro: https://github.com/dotnet/arcade/issues/14155. -->
+    <NoWarn>$(NoWarn);NU1507</NoWarn>
+  </PropertyGroup>
+
+  <!-- These versions are not updated by dependency flow as they aren't produced live anymore. -->
+  <PropertyGroup>
+    <SystemRuntimeCompilerServicesUnsafeVersion>6.0.0</SystemRuntimeCompilerServicesUnsafeVersion>
+    <SystemNumericsVectorsVersion>4.5.0</SystemNumericsVectorsVersion>

```

Comment:
We don't want properties for these versions. Just directly add them into the PackageVersion item below. That will allow dependabot to send update PRs for them in the case of a servicing release and it makes this less verbose.